Assets/style.css

```diff
@@ -0,0 +1,713 @@
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: sans-serif;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+}
+
+main section:not(:first-of-type) {
+  padding: 0 1rem;
+  max-width: 1192px;
+  margin: auto auto 4.5em auto;
+}
+
+/* Utility classes */
+.p-0 {
+  padding: 0 !important;
+}
+
+.fw-bold {
+  font-weight: bold !important;
+}
```

Comment:
Please, do not use `!important`, it is only applicable in special cases. Kindly remove it